src/components/TaskDashboard.tsx

```diff
@@ -25,23 +28,7 @@ function TaskDashboard() {
         className="todo-list stack-large stack-exception"
         aria-labelledby="list-heading"
       >
-        <li className="todo stack-small">
-          <div className="c-cb">
-            <input id="todo-0" type="checkbox" />
-            <label className="todo-label" htmlFor="todo-0">
-              Model Task
-            </label>
-          </div>
-          <div className="btn-group">
-            <button type="button" className="btn">
-              save
-              <span className="visually-hidden" />
-            </button>
-            <button type="button" className="btn btn__danger">
-              Delete <span className="visually-hidden" />
-            </button>
-          </div>
-        </li>
+        <li className="todo stack-small">{listName}</li>
```

Comment:
Não podemos colocar esse `li` aqui. 

No html, li significa `list item`. Temos que ter um `li` para cada items, e não um com todos os items dentro. Faz sentido?

O correto seria so colocar a lista e mover o `li` para dentro do ``ListModel`.
```suggestion
        {listName}
```